packages/expandable_reorderable_list/README.md

```diff
@@ -1,3 +1,70 @@
+## Expandable Reorderable List
+
 A wrapper around
-[ReorderableListView](https://api.flutter.dev/flutter/material/ReorderableListView-class.html)
-that allows you to expand and collapse items.
\ No newline at end of file
+[`ReorderableListView`](https://api.flutter.dev/flutter/material/ReorderableListView-class.html)
+that allows you to:
+- Display a tree of items.
+- Expand and collapse them.
+- Reorder them.
+
+Checkout [our example](https://github.com/Novade/flutter_packages/tree/master/packages/expandable_reorderable_list/example):
+
+
+![](https://github.com/Novade/flutter_packages/tree/master/packages/expandable_reorderable_list/resources/expandable_reorderable_list.example.gif)
+
+
+## Example
+
+This example, inspired by the [`ReorderableListView`](https://api.flutter.dev/flutter/material/ReorderableListView-class.html) documentation,
+creates a list using `ExpandableReorderableList`:
+
+```dart
+class MyList extends StatefulWidget {
+  const MyList({Key? key}) : super(key: key);
+
+  @override
+  _MyListState createState() => _MyListState();
+}
+
+class _MyListState extends State<HomeTest> {
+  final List<int> _items = List<int>.generate(50, (index) => index);
+
+  @override
+  Widget build(BuildContext context) {
+    final ColorScheme colorScheme = Theme.of(context).colorScheme;
+    final Color oddItemColor = colorScheme.primary.withOpacity(0.05);
+    final Color evenItemColor = colorScheme.primary.withOpacity(0.15);
```

Comment:
Specifying the type of variable is redundant here.

Reply:
Done in https://github.com/Novade/flutter_packages/pull/4/commits/1aaff8e1a5ca3596f08262ab9f48cae87dacac5c